sites/baseline/_screen-sm-min.scss

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
Because we have a bunch of other file in "baseline" we will need to make it easier to understand from a file system perspective.
So I propose either:
* Prefix the screen specific CSS with the component name, like "_lists-screen-sm-min.scss"; or
* Move all list related scss into a subfolder of baseline named "lists".

Reply:
> Because we have a bunch of other file in "baseline" we will need to make it easier to understand from a file system perspective. So I propose either:
> 
> * Prefix the screen specific CSS with the component name, like "_lists-screen-sm-min.scss"; or
> * Move all list related scss into a subfolder of baseline named "lists".

Added a "lists" folder and moved all lists associated styles into it.